### PR TITLE
Adding ,,conferences" section to ,,about us page" 

### PR DESCRIPTION
**Pull request by filiptaticek**

### CODE REVIEW

File: pages/about/index.tsx [hunk 1]
```diff
@@ -272,6 +272,17 @@ const About: NextPage<{ teams: TeamSection[] }> = ({ teams }) => {
         <H2>{t('about_page.media_mentions_title')}</H2>
         <ArticlesList articles={articles} />
 
+        <H2>{t('about_page.conferences_title')}</H2>
+        <Trans
+          i18nKey={'about_page.conferences_description'}
+          t={t}
+          components={[
+            <TextLink href="https://www.movapp.cz/" key="movapp_page" />,
+            <TextLink href="https://www.youtube.com/watch?v=ThY0ZiWmBV8&t=2580s" target="_blank" key="first_conference" />,
```
Comment (met) [inline pages/about/index.tsx:281]: Correct video timing https://www.youtube.com/watch?v=ThY0ZiWmBV8&t=2353s

File: pages/about/index.tsx [hunk 2]
```diff
@@ -272,6 +272,17 @@ const About: NextPage<{ teams: TeamSection[] }> = ({ teams }) => {
...
+            <TextLink href="https://www.youtube.com/watch?v=3UC16MhE19k&t=1847s" target="_blank" key="second_conference" />,
```
Comment (met) [inline pages/about/index.tsx:282]: Correct video timing https://www.youtube.com/watch?v=3UC16MhE19k&t=1660s

File: pages/about/index.tsx [hunk 3]
```diff
@@ -272,6 +272,17 @@ const About: NextPage<{ teams: TeamSection[] }> = ({ teams }) => {
         <H2>{t('about_page.media_mentions_title')}</H2>
         <ArticlesList articles={articles} />
 
+        <H2>{t('about_page.conferences_title')}</H2>
+        <Trans
+          i18nKey={'about_page.conferences_description'}
+          t={t}
+          components={[
+            <TextLink href="https://www.movapp.cz/" key="movapp_page" />,
```
Comment (met) [inline pages/about/index.tsx:280]: I believe, this link back to movapp homepage is not neccesary,

File: public/locales/cs/common.json [hunk 4]
```diff
@@ -90,6 +90,8 @@
     "why_movapp_license": "Texty, obrázky a audiosoubory Movapp jsou pod <0>licencí CC BY-NC 4.0</0>. (Uveďte původ-Neužívejte komerčně 4.0), což znamená, že je můžete pro vlastní potřeby používat zcela bez omezení. Pokud byste je chtěli dále šířit (v původní nebo v upravené podobě), můžete tak dělat pouze nekomerčním způsobem a s uvedením původních autorů.",
     "why_movapp_origin": "Původní autorkou <0>základní sady slov a frází</0> je Tereza Javornická a Yevheniia Valova (EDUA Group). Tuto sadu udržuje a dále rozšiřuje tým Movapp.",
     "media_mentions_title": "Napsali o nás",
+    "conferences_title": "Konference a přednášky",
+    "conferences_description": "Prezentace <0>Movapp.cz</0> na LUkrainian Conference na Lancaster University (Velká Británie). <br/>- <1>1st LUkrainian Conference (22. 07. 2022)</1><br/>- <2>4th LUkrainian Conference (13. 01. 2023)</2>",
```
Comment (met) [inline public/locales/cs/common.json:94]: Remove link from <0>Movapp.cz</0> 
here and in other language variants.

**Review by met — CHANGES_REQUESTED** [review]
Thanks, only small corrections here.

File: public/locales/cs/common.json (hunk 4 above)
Comment (filiptaticek) [reply public/locales/cs/common.json]: it should be fixed now, please take a look

**Review by met — APPROVED** [review]
Great, thanks.

**Pull request merged**